### PR TITLE
Bug fix - Compare method calls instead of methods themselves

### PR DESCRIPTION
**Pull request by hamidelmaazouz**

### CODE REVIEW

File: src/tests/test_qasm.py
Comment (jfriel-oqc) [inline src/tests/test_qasm.py:1357]: we want this test to start with a legacy calibration, right?

**Review by jfriel-oqc — APPROVED** [review]
approving to unblock, looks good @keriksson-rosenqvist want your approval here to move ahead

**Comment by hamidelmaazouz** [comment]
Closing this PR as it's included in another similar PR. Same story with doc build test failures

**Pull request closed without merging**
